frontend/app/controllers/Giraffe.scala

```diff
@@ -62,14 +67,27 @@ object Giraffe extends Controller {
     )
   }
 
+  def contributeRedirect = NoCacheAction { implicit request =>
+    val countryGroup = request.getFastlyCountry.getOrElse(CountryGroup.RestOfTheWorld)
+
+    val baseUrl: Uri = redirectToGiraffe(countryGroup).absoluteURL
+    val paramsToPropagate = request.queryString.map { case (k,v) => k }.toSeq
```

Comment:
It might be wise to extract the querystring concat as a function and write a unit test around it, simply so we can verify it produces valid url's with multiple query params.